.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: php
 
 php:
-  - 5.5
-  - 5.6
-  - hhvm
+  - 7.2
```

Comment:
I think the `php-7.1` is required because this version is still under official PHP support.